contracts/UtilisationRatesCalculator.sol

```diff
@@ -0,0 +1,105 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.2;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "./IRatesCalculator.sol";
+import "./WadRayMath.sol";
+
+
+/**
+ * @title UtilisationRatesCalculator
+ * @dev Contract which calculates the interest rates based on pool utilisation.
+ * Utilisation is computed as the ratio between funds borrowed and funds deposited to the pool.
+ * Rates are calculated using a linear function with slope defined by _utilisation factor and
+ * shifted by an offset parameter.
+ */
+contract UtilisationRatesCalculator is IRatesCalculator, Ownable {
```

Comment:
Let's move the unused code (this and FixedRatesCalculator) to deprecated folder, so it won't add to the audit.

Reply:
Good point

Reply:
Maybe we can do it for tests as well? we could run all test with some flag, but for usual cases we would just run the current ones (which will save a lot of time)

Reply:
Moved to deprecated in https://github.com/ava-loan/avaloan/pull/17